### ISSUE TITLE
Sabnzbd Configuration - hostname failure

### ISSUE THREAD

**Issue by Barrow1990** [sudobox-io/sb-install#9]
Server: Hetzner VPS - 2GB Hosted in Helsinki
OS: Debian 11
Error Message: Access denied - Hostname verification failed: https://sabnzbd.org/hostname-check
Requires config to be altered to allow hostname.
Specifically the host_whitelist: requires the initial line to be taken off and add "sub.domain.tld'

Default config attached from Location: /opt/sudobox/appdata/sabnzbd:
(Changed file type to .txt from .ini to be able to upload it)
[sabnzbd.txt](https://github.com/sudobox-io/sb-install/files/8240717/sabnzbd.txt)

**Comment by HawksRepos** [comment]
Due to recent(ish) updates of Sabnzbd, it is required to whitelist your domain in the sabnzbd.ini file in the appdata folder after initial installation. For SudoBox to be able to do this autonomously, we will need to run commands on the host. This will be discussed and developed in the future.

Sabnzbd link for reference:

[Sabnzbd - External internet access](https://sabnzbd.org/wiki/extra/access-denied.html#:~:text=If%20you%20try%20to%20access,a%20device%20on%20the%20internet.)

"If you try to access SABnzbd and you get External internet access denied, the cause is that you or the software/services that you are using are trying to access SABnzbd from a device on the internet. By default, because of the security reasons mentioned above, SABnzbd refuses such connections from sources on the internet."

**Comment by Barrow1990** [comment]
Agreed. Is this something that could be done on install? With a default config that's auto populated with the domain record.
